src/dirs.proj

```diff
@@ -4,6 +4,7 @@
   <ItemGroup>
     <!-- Exclude the UAP runner from the build as it depends on Win10 SDKs -->
     <ProjectsToExclude Include="xunit.runner.uap\XUnit.Runner.Uap.csproj" />
+    <ProjectsToExclude Include="xunit.runner.uap\RunnerRemoteExecutionServiceUAP\RunnerRemoteExecutionServiceUAP.csproj" />
```

Comment:
Yes. The new added project is referenced from xunit.runner.uap which is already excluded. the way we update the runner xunit.runner.uap today is by manually compiling it and packaging it. I think @AlexGhiondea used buildtools repo as a source control only and not to compile it.